src/gha_runner/__main__.py

```diff
@@ -1,13 +1,107 @@
 from gha_runner.clouddeployment import CloudDeploymentFactory
 from gha_runner.gh import GitHubInstance
-import sys
 import os
+import json
 
-# https://github.com/hukkin/tomli?tab=readme-ov-file#building-a-tomlitomllib-compatibility-layer
-if sys.version_info >= (3, 11):
-    import tomllib
-else:
-    import tomli as tomllib
+
+def env_parse_helper(
```

Comment:
let's go ahead and preface this with `_` to make it private (not that anyone is likely to import from here, but just as a matter of best practice, I don't think we intend to make this part of any API)

Reply:
Yeah I was curious on the best practice here myself. I will preface these functions.

Reply:
Fixed in [20accf4](https://github.com/omsf-eco-infra/gha-runner/pull/8/commits/20accf4687ce400398589c032476f402b4f127fd)